inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -0,0 +1,96 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Restaurant {
+
+    //Private variables that will represent the properties of a restaurant
+    private String name;
+    private String numOfDollarSigns;
+    private int numOfStars;
+    private ArrayList <Review> review = new ArrayList<>();
```

Comment:
You want to be careful about assigning values to instance variables. Since each restaurant has its own ArrayList of reviews it would've been better to assign this empty list in the constructor

Reply:
So you mean like declaring the arrayList inside of my constructor?